Islandora-Devops/islandora-playbook
# Fits

**GitHub Issue**: (link)

# What does this Pull Request do?

This PR installs the work for FITS media that @ajstanley has developed for UPEI's RDM project.  Now we can test it :tada: :rocket:

# What's new?

Points to a dev version of the fits role and adds that role back into the tomcat play so it gets brought in when provisioning.

# How should this be tested?

Make a node, add an original file to the node.  A FITS media derivative should get made, and you can see all the metadata that gets extracted from it.


# Interested parties
@Islandora-Devops/committers @ajstanley @elizoller
